### ISSUE TITLE
Enhancement: Add a footer and scroll to top button

### ISSUE THREAD

**Issue by Preeti8021** [ShwetKhatri2001/Swetify-Music#23]
I have seen the website is missing the footer section and would like to add a footer to the webpage which would be having the copyright and links to above sections. Additionally, I want to add a scroll to top button.

**Comment by Preeti8021** [comment]
Please assign this task to me under CodePeak'23

**Comment by ShwetKhatri2001** [comment]
Sure, you can start working on adding both. Make sure the footer contains the links present on our navbar .
Also, you can add some more links that you think this kind of application should have. The extra links will direct to home page only till we create those pages.

**Comment by Preeti8021** [comment]
yes, sure !

**Comment by Preeti8021** [comment]
Kindly review the changes. If everything is ok, I will send a pull request. 

https://github.com/ShwetKhatri2001/Swetify-Music/assets/103981488/8bae6299-5d8e-49a5-8a01-2df689dee1f3

**Comment by ShwetKhatri2001** [comment]
Yes, it's good. Make it responsive for all viewport widths if not done yet.

**Comment by Preeti8021** [comment]
Yes, the responsiveness is done.
